src/sgb-xi/codes.ts

```diff
@@ -0,0 +1,289 @@
+
```

Comment:
I'd like to suggest to add a comment that references the source on which the implementation is based. I noticed that new versions of those documents are posted every 0.5 to 2 years or so, so it would be good to be able to see later if the implementation needs updating or not.
And of course, to include the reference is also good for documentation, since you already reference the Kapitelnummern of the different Schlüssel.

For the Kostenträger parser, I for example added this comment at the beginning of the file, I included both the title etc and version printed on the first page and the link, in case the link does not work anymore some time in the future:

```typescript
/** 
 *  Parses a Kostenträgerdatei (payer file) which provides information how to send invoices to the
 *  statutory health insurance as a health care service provider.
 * 
 *  The implementation is based on ....
 *  
 *  Anhang 5 zur Anlage 1 - Kostenträgerdatei,
 *  Regelung der Datenübermittlung nach § 105 Abs. 2 SGB XI Technische Anlage (Anlage 1),
 *  Version 3.3, Effective starting 01.10.2021
 *  
 *  https://gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/pflege/technische_anlagen_aktuell_2/TA1_ANH5_20170907_105_oA.pdf
 * 
 *  ...and thus only covers Pflegedienstleistungen (care providers)
 */
```

Reply:
created docs/documents.md as central documentation sources overview